### ISSUE TITLE
Q: dc:date vs oio:date_created for new terms?

### ISSUE THREAD

**Issue by matentzn** [OBOFoundry/OBOFoundry.github.io#906]
As part of a [discussion on annotation properties](https://github.com/obophenotype/human-phenotype-ontology/issues/4885) (HP and MP) we were wondering whether we (OBO foundry) have an opinion on whether we should be using

http://www.geneontology.org/formats/oboInOwl#creation_date

or 

http://purl.org/dc/elements/1.1/date

to denote the creation date of a new term, or whether we have no opinion :) Thanks.

**Comment by nlharris** [comment]
can anyone answer nico's question?

**Comment by matentzn** [comment]
100% dc:date

**Comment by nlharris** [comment]
apparently nico answered his own question. can we close this issue?

**Comment by dosumis** [comment]
AFAIK, everything coming from the OBO world uses 

http://www.geneontology.org/formats/oboInOwl#creation_date

May be a slow job getting everyone to change.

**Comment by matentzn** [comment]
Yeah, unfortunately.. I will mention it at the obo operations committee next time.

**Comment by cthoyt** [comment]
I also vote for dc:date

**Comment by zhengj2007** [comment]
I vote for dc:date too.
